subscription-manager.spec

```diff
@@ -1,31 +1,22 @@
-# Prefer systemd over sysv on Fedora 17+ and RHEL 7+
-%global use_systemd (0%{?fedora} && 0%{?fedora} >= 17) || (0%{?rhel} && 0%{?rhel} >= 7) || (0%{?suse_version} && 0%{?suse_version} >= 1315)
+# Prefer systemd over sysv on Fedora and RHEL 7+
+%global use_systemd 0%{?fedora} || (0%{?rhel} && 0%{?rhel} >= 7) || (0%{?suse_version} && 0%{?suse_version} >= 1315)
 # For optional building of ostree-plugin sub package. Unrelated to systemd
 # but the same versions apply at the moment.
 %global has_ostree %use_systemd && 0%{?suse_version} == 0
-%global use_firstboot 0
 %global use_initial_setup 1
+%global use_firstboot 0
+%global use_kitchen 1
 %global rhsm_plugins_dir  /usr/share/rhsm-plugins
 %global use_gtk3 %use_systemd
 
-%if 0%{?rhel} == 7
-%global use_initial_setup 1
-%global use_firstboot 0
-%endif
-
-# 6 < rhel < 7
-%if 0%{?rhel} == 6
-%global use_initial_setup 0
-%global use_firstboot 1
-%endif
-
-# SLES
-%if 0%{?suse_version}
+%if 0%{?rhel} == 6 || 0%{?suse_version}
 %global use_initial_setup 0
 %global use_firstboot 1
+%global use_kitchen 0
 %endif
 
-%global use_dnf (0%{?fedora} && 0%{?fedora} >= 22)
+%global use_dnf 0%{?fedora}
+%global use_yum (0%{?rhel} && 0%{?rhel} <= 7)
```

Comment:
Since SUSE needs python-yum for the `utf8_width`, etc. for now at least, we should assume that SUSE builds use yum too.